src/modm/board/disco_f411ve/board.hpp

```diff
@@ -22,13 +24,53 @@ namespace Board
 /// @{
 using namespace modm::literals;
 
-/// STM32F411 running at 96MHz generated from the external 8MHz crystal
+
 struct SystemClock
 {
-	static constexpr uint32_t Frequency = 96_MHz;
-	static constexpr uint32_t Ahb = Frequency;
-	static constexpr uint32_t Apb1 = Frequency / 2;
-	static constexpr uint32_t Apb2 = Frequency;
+	// static constexpr uint32_t ExternalLSEclock = ;
+	static constexpr uint32_t ExternalHSEclock = 8_MHz;
+
+	static constexpr Rcc::PllFactors pllFactors{
+		.pllM = 8,
+		.pllN = 336,
+		.pllP = 4,
+		.pllQ = 7,
+	};
```

Comment:
I've temporary switched the pll factors from
M=7, N=366, P=4, Q=8 to
M=**8**, N=366, P=4, Q=**7**
for cleaner intermediate clocks.